### PR TITLE
Bulgarian language

### PR DESCRIPTION
**Pull request by michal-pawlowski**

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/FluentValidation/FluentValidation?pullRequest=1973) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/FluentValidation/FluentValidation?pullRequest=1973) before we can accept your contribution.<br/><br/>:x: michal-pawlowski [sign now](https://cla.dotnetfoundation.org/FluentValidation/FluentValidation?pullRequest=1973)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/FluentValidation/FluentValidation?pullRequest=1973) it.</sub>

File: src/FluentValidation/Resources/Languages/BulgarianLanguage.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+//
+// The latest version of this file can be found at https://github.com/FluentValidation/FluentValidation
+
+#endregion
+
+#pragma warning disable 618
+
+namespace FluentValidation.Resources;
+
+internal class EnglishLanguage {
```
Comment (JeremySkinner) [inline src/FluentValidation/Resources/Languages/BulgarianLanguage.cs:25]: Hi @michal-pawlowski could you fix the class name? It should be `BulgarianLanguage`

Comment (michal-pawlowski) [reply src/FluentValidation/Resources/Languages/BulgarianLanguage.cs]: Sorry, my bad. Fixed.

**Comment by JeremySkinner** [comment]
Merged, thanks!

**Pull request merged**